bundle/config/mutator/resourcemutator/resource_mutator.go

```diff
@@ -0,0 +1,311 @@
+package resourcemutator
+
+import (
+	"context"
+	"errors"
+
+	"github.com/databricks/cli/bundle/config/mutator"
+
+	"github.com/databricks/cli/libs/dyn/merge"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/dyn"
+)
+
+// When a new resource is added to configuration, we apply bundle
+// settings and defaults to it. Initialization is applied only once.
+//
+// If bundle is modified outside of 'resources' section, these changes are discarded.
+func initializeMutators() []bundle.Mutator {
```

Comment:
We've moved away from defining chains of mutators in favor of defining eager functions that perform the necessary actions (see https://github.com/databricks/terraform-provider-databricks/pull/2390)  - much easier to work with.

So you can just do ApplyInitializeMutators(ctx, bundle) here.

Reply:
I don't get it, is the link correct?